### PR TITLE
Boilerplate

### PR DESCRIPTION
**Pull request by WenxuC**

### CODE REVIEW

File: server/server.js [hunk 1]
```diff
@@ -1,37 +1,36 @@
 const express = require('express');
-const cors = require('cors');
+//const cors = require('cors');
```
Comment (WenxuC) [inline server/server.js:2]: do we need this? and line 10

File: server/server.js [hunk 2]
```diff
@@ -1,37 +1,36 @@
...
+
 const mongoose = require('mongoose');
-// const parseImage = require('./visionAPI');
-require('dotenv').config({ path: '../.env' });
-const dbKey = process.env.dbKey;
+require('dotenv').config();
```
Comment (WenxuC) [inline server/server.js:5]: the problem was the path, it returned undefined, I console logged the new code and was able to get the key

**Review by channnx — APPROVED** [review]

File: server/server.js (hunk 1 above)
Comment (gabzn) [reply server/server.js:2]: We do need this for development. You cannot get data from backend without it.

File: server/server.js (hunk 2 above)
Comment (gabzn) [reply server/server.js:5]: This one I need to test out tomorrow. The path shouldn't be an issue.

**Pull request merged**